splunk-quarkus/clowdapp.yaml

```diff
@@ -83,3 +84,12 @@ parameters:
   - name: KAFKA_INGRESS_TOPIC
     description: Kafka topic that the consumer takes messages from
     value: platform.notifications.tocamel
+  - name: CLOWDER_ENABLED
```

Comment:
`CLOWDER_ENABLED`, this one looks like it was brought from Drift. What it supposed to do? If it needs to be mapped to env vars then it needs to be within `podSpec`.